layouts/partials/visualization.html

```diff
@@ -50,6 +50,12 @@
                             align="middle">
                     </a>
                 </div>
+                <div>
+                    <a href="https://docs.pyvista.org/examples/index.html">
+                        <img src="images/content_images/v_pyvista.png" alt="A 3D volume rendering made in PyVista."
+                            align="middle">
```

Comment:
This `img` element is missing the closing `/`, so you'll want it like `... align="middle" />`

Reply:
Ah, my bad 🤦‍♂️ - will fix!

Reply:
I just noticed none of the `img` elements on that page have a closing `/` so I added one for all